examples/custom-grpc-network/README.md

```diff
@@ -0,0 +1,97 @@
+# Custom Network Configuration with gRPC Web Proxies
+
+This guide demonstrates how to configure the Hedera SDK to communicate with a custom network using gRPC web proxies in a React application. The example sets up a transfer transaction, which is triggered when the page is loaded, using a custom network configuration with multiple gRPC web proxies.
+
+## Steps to Configure the Custom Network
+
+### 1. Setup the Operator Account
+
+First, you need to set up your operator account and private key. These credentials are required to sign and authorize the transaction.
+
+```javascript
+const operatorId = AccountId.fromString("0.0.1458");
+const operatorKey = PrivateKey.fromStringECDSA(
+    "3030020100300706052b8104000a042204208c4ffe487636e8102f72502a2ed16cb4f98073582a777fcfa758481485d9f887",
+);
+```
+
+Replace `0.0.1458` and the private key with your own operator account ID and key.
+
+### 2. Create a Custom Network with gRPC Web Proxies
+
+Define a list of gRPC web proxies to communicate with your custom network. Each entry consists of a proxy URL and its associated account ID.
+
+```javascript
+const nodes = {
+    "https://testnet-node02-00-grpc.hedera.com:443": new AccountId(5),
+    "https://testnet-node03-00-grpc.hedera.com:443": new AccountId(6),
+    "https://testnet-node04-00-grpc.hedera.com:443": new AccountId(7),
+};
+```
+
+These proxy URLs correspond to the gRPC web proxies you wish to use for the custom network.
+
+### 3. Setup the Client Using Client.forNetwork()
+
+To configure the client with the custom network, use `Client.forNetwork()` and pass the list of nodes you defined. The SDK automatically detects the environment (whether it's a browser or Node.js environment) and selects the appropriate implementation (NodeClient, WebClient, or NativeClient).
```

Comment:
```suggestion
To configure the client with the custom network, use `Client.forNetwork()` and pass the list of nodes you defined. The SDK automatically detects the environment (whether it's a browser or Node.js environment).
```

I don't think there's any need to add such details, that we have three different implementations since it's an example and it's main idea is how to be used and not very deep technical details